cost-analyzer/templates/cost-analyzer-ingestion-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if eq (include "aggregator.deployMethod" .) "statefulset" }}
+{{- if .Values.kubecostProductConfigs.standardDiscount }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ default "ingestion-configs" .Values.ingestionConfigmapName }}
```

Comment:
Thoughts on changing `.Values.ingestionConfigmapName` to something like `.Values.kubecostProductConfigs.standardDiscountConfigmapName`?

Reply:
The idea was (ref discussion with Alex in: https://github.com/kubecost/kubecost-cost-model/pull/2668) to have a specific dedicated configmap + config struct for all ingestion-level config items (Obviously this discount is the only one present currently, but for posterity's sake). So I think `ingestionConfigmap` is correct for the intent, at least.

Reply:
Ok, good with me!